### PR TITLE
DFAST-212: Review updated reporting of conditions

### PR DESCRIPTION
**Pull request by MRVermeulenDeltares**

### CODE REVIEW

File: dfastmi/batch/core.py [hunk 1]
```diff
@@ -260,22 +260,38 @@ def _report_used_file_names(
...
             )
+        )
+
+        _report_analysis_conditions_values(
+            condition, reference_file_name, measure_file_name, comment, report
+        )
+
+
+def get_analysis_condition_values_for_logging(
+    initialized_config: AConfigurationInitializerBase, filenames, q, key
```
Comment (hrajagers) [inline dfastmi/batch/core.py:276]: Add typing

**Comment by hrajagers** [comment]
Agree with this change. It's cleaner. We may have to change this code again once the tool gets used because I'm not sure that reporting just the file names is enough. It's likely that all filenames will be equal in daily practice and only the full paths are different. However, that's for later concern.

File: dfastmi/batch/core.py (hunk 1 above)
Comment (MRVermeulenDeltares) [reply dfastmi/batch/core.py:276]: Will do!

Comment (MRVermeulenDeltares) [reply dfastmi/batch/core.py]: Done

**Pull request merged**